src/main/scala/com/fulcrumgenomics/vcf/HapCutToVcf.scala

```diff
@@ -117,6 +123,32 @@ class HapCutToVcf
     vcfWriter.close()
   }
 
+  /** If fixing ambiguous (IUPAC) bases is desired, returns a new path to a VCF, otherwise returns the original input VCF. */
+  private def inputVcf: FilePath = if (!fixAmbiguousReferenceAlleles) this.vcf else {
+
+    val inputLines = PathUtil.extensionOf(this.vcf) match {
+      case Some(".vcf") => Io.readLines(this.vcf)
+      case Some(".vcf.gz")  =>
+        val stream = Io.toInputStream(this.vcf)
```

Comment:
Doesn't `Io.readLines` also work for gzipped files?  If not we should make it, because it _should_.

Reply:
Fixed